spec/features/viewing_peeps_spec.rb

```diff
@@ -0,0 +1,9 @@
+require_relative '../setup_test_database'
+
+feature 'viewing peeps' do
+  scenario 'the user can view peeps on the homepage with the date posted' do
+    add_row_to_test_database
+    visit('/')
+    expect(page).to have_content "This is a peep! Posted on 2022-04-01."
+  end
```

Comment:
You can add a test for reverse chronological order. Check how you can use capybara matchers to match a regular expression.